ci_build/steps_template_real_model_test.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  python_version: '3.5'
+  tensorflow_version: ''
+  opset_version: ''
+  onnx_version: ''
+  onnxruntime_version: ''
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '${{ parameters.python_version }}'
+    architecture: 'x64'
+    displayName: 'Set python version'
+
+- bash: |
+    python -m pip install --upgrade pip setuptools wheel
+    pip install pytest pytest-cov pytest-runner graphviz pylint requests pyyaml Pillow
+    pip install tensorflow==${{ parameters.tensorflow_version }}
+    if [ "${{ parameters.onnx_version }}" == "" ]; then
+      pip install onnx
+    else
+      pip install onnx==${{ parameters.onnx_version }}
+    fi
+
+    if [ "${{ parameters.onnxruntime_version }}" == "" ]; then
+      pip install onnxruntime
+    else
+      pip install onnxruntime==${{ parameters.onnxruntime_version }}
+    fi
+    # install onnx converter-"tf2onnx"
+    python setup.py install
+    pip list
+  displayName: 'Install dependencies'
+
+- bash:
+    pylint --rcfile=tools/pylintrc --ignore=version.py --disable=cyclic-import tf2onnx tests/*.py tools
+  continueOnError: true
```

Comment:
we still want this build to fail. Does it do this?

Reply:
azure pipeline will report the status as "success with issue".

reason why write it in this way is that i think maybe it can save test time to run pylint test and computation test

Reply:
I just worried that we accidentally merge with pylint errors. Up to you. We can always change it if this becomes an issue.